android/app/build.gradle

```diff
@@ -1,9 +1,10 @@
-apply plugin: 'com.android.application'
+apply plugin: "com.android.application"
+apply plugin: "com.facebook.react"
```

Comment:
I think this might be the reason for failures on Android. We don't use the RN plugin because it's not very friendly to libraries (our SDK is a library). THis might have changed because I see it's written in Kotlin now, but I reckon we could do the update without it, and reconsider later.

Reply:
Another important point: this is the app gradle file. We don't use RN at all in the app, just the SDK.